### ISSUE TITLE
Two issues on Participants

### ISSUE THREAD

**Issue by clapierre** [w3c/graphics-aam#2]
B.1 Participants active in the SVG accessibility task force at the time of publication 
Doug Schepers (W3C Staff)

First SVG I believe should be changed to Graphics Accessibility...
And Doug hasn't been with the W3C for over a year now.
Thanks

**Comment by michael-n-cooper** [comment]
The Task Force was named SVG Accessibility, so that is correct in the document. https://www.w3.org/WAI/PF/svg-a11y-tf/

Acknowledgements have been cleaned up via https://github.com/w3c/graphics-aam/commit/7a23f49708d7ad8edd0c12b31a2230b600b7ef7f. Doug and others are still credited, as task force participants as the document is in substantially the same form it was when these participants were last active.
